enterprise/server/backends/migration_cache/migration_cache.go

```diff
@@ -834,20 +836,26 @@ func (mc *MigrationCache) copyDataInBackground() error {
 	}
 }
 
-func (mc *MigrationCache) logCopyChanFullInBackground() {
-	ticker := time.NewTicker(mc.copyChanFullWarningInterval)
-	defer ticker.Stop()
+func (mc *MigrationCache) logCopyChanInfoInBackground() {
```

Comment:
what do you think about calling this `monitorCopyChanFullness`?

Reply:
Like that - fixed